secrets.py

```diff
@@ -0,0 +1,4 @@
+import os
```

Comment:
see my ideas on telegram

Reply:
made the changes we discussed in 545d4a2 - that look good?

Reply:
yep, looks good